src/mon/MonCommands.h

```diff
@@ -331,7 +331,8 @@ COMMAND("fs new "
 	" name=metadata,type=CephString "
 	"name=data,type=CephString "
 	"name=force,type=CephBool,req=false "
-	"name=allow_dangerous_metadata_overlay,type=CephBool,req=false",
+	"name=allow_dangerous_metadata_overlay,type=CephBool,req=false "
+	"name=fs_id,type=CephInt,range=0,req=false",
```

Comment:
Should set the min range to 1 instead of zero?

Reply:
No, really old cephfs file systems are usually `fscid==0`.

Reply:
See LRC:

````
e747732
enable_multiple, ever_enabled_multiple: 1,1
compat: compat={},rocompat={},incompat={1=base v0.20,2=client writeable ranges,3=default file layouts on dirs,4=dir inode in separate object,5=mds uses versioned encoding,6=dirfrag is stored 
in omap,8=no anchor table,9=file layout v2,10=snaprealm v2}
legacy client fscid: 0
 
Filesystem 'cephfs' (0)
fs_name cephfs
epoch   746957
flags   14
created 2013-08-14T10:19:58.297184+0000
modified        2021-06-16T17:43:33.655898+0000
tableserver     0
root    0
session_timeout 60
session_autoclose       300
max_file_size   1099511627776
required_client_features        {}
last_failure    82060
last_failure_osd_epoch  10587310
compat  compat={},rocompat={},incompat={1=base v0.20,2=client writeable ranges,3=default file layouts on dirs,4=dir inode in separate object,5=mds uses versioned encoding,6=dirfrag is stored in omap,8=no anchor table,9=file layout v2,10=snaprealm v2}
max_mds 1
in      0
up      {0=827742276}
failed  
damaged 
stopped 
data_pools      [0]
metadata_pool   1
inline_data     disabled
balancer        
standby_count_wanted    1
[mds.cephfs.reesi001.umftpx{0:827742276} state up:active seq 22 join_fscid=0 addr [v2:172.21.2.201:6800/729508947,v1:172.21.2.201:6801/729508947]]
````

Reply:
Got it! Thanks!

Reply:
And I need to hide the option to create a file system with a specific ID behind --force flag